cypress/e2e/company-signup.cy.ts

```diff
@@ -0,0 +1,107 @@
+describe("company signup", () => {
+  it("Company signup flow", () => {
+    // Visit company page
+    cy.visit("/accounts/signup");
+
+    // Click on company login choice
+    cy.contains("I want to assess potential employees")
+      .parent()
+      .click()
+      .should("have.css", "box-shadow", "rgb(144, 118, 192) 0px 0px 0px 2px");
+
+    // Continue to next page and check if it actually redirects
+    cy.contains("Next").click();
+
+    cy.contains("Your details").should("exist");
+
+    // Error when directly submit
+    cy.get("button").click();
+
+    cy.contains("Please fill in this field.").should("exist");
+
+    // Fill in the fields and submit
+    cy.get("input").type("PT. Minterm Indonesia");
```

Comment:
Hello Alex, from my understanding, this will get the first input element it finds. Is there any way to get the specific input field so the test won't break when requirements change? Let me know 🙂

Reply:
I haven't figured it out yet Rashad. Because of the CSS module, the class name will be prefixed and suffixed, which means it's difficult to select it based on the class name.